src/nbl/builtin/template/CArchive.h.in

```diff
@@ -1,11 +1,12 @@
 #ifndef _@_GUARD_SUFFIX_@_C_ARCHIVE_H_
 #define _@_GUARD_SUFFIX_@_C_ARCHIVE_H_
 
-#include <nbl/system/CFileArchive.h>
+#include "nbl/system/CFileArchive.h"
 #include "@NBL_BS_HEADER_FILENAME@"
 
 namespace @_NAMESPACE_@
 {
+    constexpr std::string_view path = "@_NAMESPACE_@/@_PATH_PREFIX_@"
```

Comment:
no need to pollute the namespace with more global variables just construct it on the fly

Also doesn't this make it `nbl::builitin/nbl/builtin/` or something weird like that!?

Reply:
corrected